github/util.py

```diff
@@ -498,6 +498,10 @@ def is_org_member(self, organization_name, user_login):
         organization = self.github.organization(organization_name)
         return organization.is_member(user_login)
 
+    def fetch_labels_on_pr(self, pull_request_number):
```

Comment:
nit: type hint?

Reply:
On further thought, I agree. I've removed it, as the benefit is miniscule